scripts/add_translation.py

```diff
@@ -0,0 +1,160 @@
+    if not credentials or credentials.invalid:
+        flow = client.flow_from_clientsecrets(CLIENT_SECRET_FILE, SCOPES)
+        flow.user_agent = APPLICATION_NAME
+        if flags:
+            credentials = tools.run_flow(flow, store, flags)
+        else:  # Needed only for compatibility with Python 2.6
+            credentials = tools.run(flow, store)
+        print('Storing credentials to ' + credential_path)
+    return credentials
+
+
+def display_languages(service):
+    print 'List of available languages:\n----------------------------'
+    data = service.files().list(q="'" + TRANSLATION_FOLDER_ID + "'" + " in parents").execute()
+    for f in data['files']:
+        if f['mimeType'].endswith('.folder'):
+            LANGUAGE_LIST[f['name']] = f['id']
+
+    # sort the languages alphabetically
+    sorted_list = [e for e in sorted(LANGUAGE_LIST)]
+    id = 1
+    # store the generated IDs to match against user input.
+    language_id = {}
+    for i in sorted_list:
+        print  str(id) + ' ' + i
+        language_id[id] = i
+        id = id + 1
+
+    response = raw_input('Select language from the list: ')
+    user_choice = language_id[int(response)]
+    print 'User selected '+user_choice+ ' language'
+    if user_choice not in LANGUAGE_LIST:
+        print '\nInvalid selection'
+        exit(1)
+
+    else:
+        return LANGUAGE_LIST[user_choice]
+
+
+def main():
+    credentials = get_credentials()
+    http = credentials.authorize(httplib2.Http())
+    service = discovery.build('drive', 'v3', http=http)
+    selection = display_languages(service)
+
+    while True:
+        # get and download files
+        data = service.files().list(q="'" + selection + "'" + " in parents").execute()
+        download_translation_file(data['files'], service)
+        response = raw_input('Add new language? (y/n)\n')
+        if response == 'y':
+            selection = display_languages(service)
+            continue
+        elif response == 'n':
+            print 'Exit..'
+            sys.exit()
+
+def download_consent_files(files, service):
+    for f in files:
+        file_id = f['id']
+        request = service.files().export_media(fileId=file_id, mimeType='text/csv').execute()
+        fn = '%s.csv' % ('consent_forms/' + f['name'].replace(' ', '_'))
+        if request:
+            with open(fn, 'wb') as csvfile:
+                csvfile.write(request)
+
+
+def download_translation_file(files, service):
+    for f in files:
+        file_id= f['id']
+        file_type= f['mimeType']
+        file_name= f['name']
+        if file_type.endswith('.spreadsheet'):
+            request = service.files().export_media(fileId=file_id, mimeType='text/csv').execute()
+            fn = '%s.csv' % (file_name.replace(' ', '_'))
+            if request:
+                with open(fn, 'wb') as csvfile:
+                    csvfile.write(request)
+
+            os.system('python format_translations.py --filename {f}'.format(f=fn))
```

Comment:
As discussed, please do not use `os.system` to call a script. See example sent via flowdock for an alternate pattern that lets you use both argparse *and* still call the module as a function from somewhere else.